lib/streamlit/util.py

```diff
@@ -106,13 +106,6 @@ def _open_browser_with_command(command, url):
         subprocess.Popen(cmd_line, stdout=devnull, stderr=subprocess.STDOUT)
 
 
-def _maybe_tuple_to_list(item: Any) -> Any:
```

Comment:
Is there a specific reason moving it to component_arrow? I feel like this could belong here.

Reply:
`type_utils` would probably be a better place for the function. But since it's only used by `component_arrow` I extracted it from utils.

Reply:
Ah yeah that's a good point. Can we just move it to `type_utils`? Going to hit approval since that's a small nit.

Reply:
done 👍

Reply:
oh, wait. This PR is actually not supposed to change the `component_arrow`, this actually needs to be in another PR.